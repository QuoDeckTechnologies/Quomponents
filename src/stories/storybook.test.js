// import initStoryshots from "@storybook/addon-storyshots";
// initStoryshots();

it("should be updated later", () => {
    expect(true).toBe(true);
});
