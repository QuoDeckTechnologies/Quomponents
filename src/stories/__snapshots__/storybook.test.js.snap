// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Design System/Badges/Badge Icon 1`] = `
<div>
  <div
    style={
      Object {
        "margin": "0 auto",
        "width": "280px",
      }
    }
  >
    <span
      alignhorizontal="right"
      alignvertical="top"
      className="MuiBadge-root css-1c32n2y-MuiBadge-root"
      content=""
      number={0}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-159ivpg-MuiSvgIcon-root"
        data-testid="MailIcon"
        focusable="false"
        onClick={[Function]}
        viewBox="0 0 24 24"
      >
        <path
          d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4-8 5-8-5V6l8 5 8-5v2z"
        />
      </svg>
      <span
        className="MuiBadge-colorPrimary MuiBadge-badge MuiBadge-standard MuiBadge-anchorOriginTopRightRectangular css-106c1u2-MuiBadge-badge"
      >
        50+
      </span>
    </span>
  </div>
</div>
`;

exports[`Storyshots Design System/Badges/Badge Text 1`] = `
<div>
  <div
    style={
      Object {
        "margin": "0 auto",
        "width": "280px",
      }
    }
  >
    <span
      alignhorizontal="right"
      alignvertical="top"
      className="MuiBadge-root css-1c32n2y-MuiBadge-root"
      content=""
      number={0}
    >
      Notifications
      <span
        className="MuiBadge-colorSuccess MuiBadge-badge MuiBadge-standard MuiBadge-anchorOriginTopRightRectangular css-pwyk3z-MuiBadge-badge"
      >
        5
      </span>
    </span>
  </div>
</div>
`;
